src/operator/every.ts

```diff
@@ -18,22 +16,21 @@ export function every<T>(predicate: (value: T, index: number, source: Observable
                          thisArg?: any): Observable<boolean> {
   const source = this;
   if (source._isScalar) {
-    const result: boolean = tryCatch(predicate).call(thisArg || this, source.value, 0, source);
-    if (result === errorObject) {
-      return new ErrorObservable(errorObject.e, source.scheduler);
-    } else {
+    try {
```

Comment:
In this operator, we have performance improvement only for the common case (non-scalar/array). Both Removing `tryCatch()` from the particular cases only, and removing it plus moving the try/catch block to separate functions changes almost nothing in terms of performance. 

What should we do in cases such as this one? Remove the tryCatch() anyway or leave as it is now?